src/servers/build.rs

```diff
@@ -14,9 +14,8 @@
 
 fn main() {
     build_data::set_RUSTC_VERSION();
-    build_data::set_GIT_BRANCH();
-    build_data::set_GIT_COMMIT();
     build_data::set_SOURCE_TIMESTAMP();
```

Comment:
This line of code use `git` and `unwrap` too, please fix this 🥲

Reply:
🤣

Reply:
Fixed.